.travis.yml

```diff
@@ -29,11 +29,13 @@ matrix:
       gemfile: gemfiles/multi_json.gemfile
       script:
         - bundle exec rake
+        - bundle exec rspec spec/integration/eager_load
         - bundle exec rspec spec/integration/multi_json
     - rvm: 2.7.1
       gemfile: gemfiles/multi_xml.gemfile
       script:
         - bundle exec rake
+        - bundle exec rspec spec/integration/eager_load
         - bundle exec rspec spec/integration/multi_xml
```

Comment:
Let's not mix multi_json and multi_xml tests with eager loading tests. Add a new section to use the default gemfile and run `rspec spec/integration/eager_load`?